src/meta/src/rpc/election_client.rs

```diff
@@ -145,16 +145,23 @@ impl ElectionClient for EtcdElectionClient {
             };
 
             let mut ticker = time::interval(Duration::from_secs(1));
+            let mut timeout = time::interval(Duration::from_secs((ttl / 2) as u64));
```

Comment:
Better to add some comment for `timeout`. Add it's necessary to add some check for `meta_leader_lease_secs` when meta starts.